icons/vsphere.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>vSphere</title><path d="M19.98 0h-6.524l-1.004 1h7.428c1.807 0 3.213 1.4 3.213 3.1v15.8c0 1.7-1.406 3.1-3.112 3.1h-6.325l-1.003 1h7.227c2.31 0 4.116-1.8 4.116-4.1V4.1c.1-2.3-1.707-4.1-4.015-4.1zm-8.432 23h-7.43c-1.708 0-3.114-1.4-3.114-3.1V4.1C1.004 2.4 2.41 1 4.117 1h6.327l1.004-1h-7.33C1.807 0 0 1.8 0 4.1v15.8C0 22.2 1.808 24 4.117 24h6.527zm7.732-1.8H8.837c-1.004 0-1.908-.8-1.908-1.9V8.6c0-1 .803-1.9 1.908-1.9H19.28c1.005 0 1.908.8 1.908 1.9v10.7c0 1-.803 1.9-1.908 1.9zM8.837 7.7c-.502 0-.904.4-.904.9v10.7c0 .5.402.9.904.9H19.28c.502 0 .904-.4.904-.9V8.6c0-.5-.402-.9-.904-.9zm-2.812 8.6H4.72a.898.898 0 01-.904-.9V4.7c0-.5.402-.9.904-.9h10.443c.502 0 .904.4.904.9v1.1h1.004V4.7c0-1-.803-1.9-1.908-1.9H4.72c-1.005 0-1.908.8-1.908 1.9v10.7c0 1 .803 1.9 1.908 1.9h1.305z"/></svg>
```

Comment:
Do you have a source for us to compare this SVG against?

Reply:
Yes, I do. Let me know how you wish to access the files.

Ryan

Reply:
Attaching the [originating](github.com/simple-icons/simple-icons/files/6912024/vmw-prod-ico-vsphere-rgb_400px.svg.zip) `.svg` file for the vSphere product icon from the [VMware Brand Central](https://www.vmware.com/brand.html).

Ryan

Reply:
Not sure what happened there but the "originating" link doesn't work for me :confused:

Reply:
I updated the link in the comment. :)

Reply:
Thank you :smiley: 

Comparing these two SVGs in Inkscape reveals some differences (see below), while this could be due to SVGO(MG) I feel like the differences are too significant given that you used a precision of `3`(?)... It could be that you did something wrong when converting the source into a single path. Could you give it another try? If you share the graphics editor you use we might be able to provide some guidance.

Also, since you're a VMware representative we _could_ go with the current version if that is what VMware prefers.

![SVG comparison](https://user-images.githubusercontent.com/3742559/127775951-629aee4b-799b-4e6a-9de8-3ef1f904e57f.png)

Reply:
Thanks! I use Adobe Illustrator. Any recommendations to confirm the vSphere icon to the simpleicons requirements would be appreciated.

Ryan

Reply:
Unfortunately I no longer have access to Illustrator but if memory serves you should look in the `Object` dropdown menu for something named `Compound path`. That should convert everything to a path, you may still need to combine the various paths. If you can't find exactly this option, or this doesn't work as expected, I'd recommend to work on a copy of the SVG on the same artboard so you can easily check if what you did distorts the image.

Alternatively, I believe @service-paradis is still using Illustrator so he might be able to help.

Reply:
> Alternatively, I believe @service-paradis is still using Illustrator so he might be able to help.

Most of the time I'm using Illustrator is to convert `.eps` or `.ai` files to SVG (I also use a french version, so menu items are not named the same way). Then I use Inkscape 😅 to do most of the transformation needed. So I might not be the best one to help with Illustrator unfortunately.

Reply:
This time it looks like you were to aggressive with the optimization, it looks like a precision of 1 or 2 which is almost always too low (unless the shapes are super simple). Also, probably due to the optimization as well, this time the SVG is `24.004x24.000`.

![comparison](https://user-images.githubusercontent.com/3742559/128603887-34cba9fa-f8b4-46a8-a4f5-2ccaf60af0f6.png)

Reply:
🤦🏻‍♂️.

Reply:
I've closed the pull request since I'm unable to get it just right.